components/popover_list_members/popover_list_members.jsx

```diff
@@ -126,7 +126,7 @@ export default class PopoverListMembers extends React.Component {
 
     handleGetProfilesInChannel = (e) => {
         this.setState({popoverTarget: e.target, showPopover: !this.state.showPopover});
-        this.props.actions.getProfilesInChannel(this.props.channel.id, 0);
+        this.props.actions.getProfilesInChannel(this.props.channel.id, 0, undefined, 'status');
```

Comment:
Instead of `undefined`, why not use the intended per page value or maybe `PER_PAGE_DEFAULT` of 60?

Reply:
I'm pretty sure the use of `undefined` is causing the PR to fail

Reply:
I actually think that is somewhat dumb. `undefined` is how you skip an attribute in an overloaded function. I am now exporting `PER_PAGE_DEFAULT` from Client4 in redux and using that, but this smells bad.
Anyway, tests should pass now.